hack/build.sh.d/kn-plugin-func.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# There no midstream image currently, fallback to upstream one.
+readonly socat_image="${KN_PLUGIN_FUNC_SOCAT_IMAGE:-quay.io/boson/alpine-socat:1.7.4.3-r1-non-root}"
+export EXTERNAL_LD_FLAGS="${EXTERNAL_LD_FLAGS:-} \
+-X knative.dev/kn-plugin-func/k8s.SocatImage=${socat_image} "
```

Comment:
That's a productization hook. It's fine to keep upstream image here as default. It's overriden during prod build.

Reply:
@ederign FYI ^

Reply:
@dsimansk, thanks for the detailed instructions. As @ljmotta will be on PTO, @paulovmr or I will work on that.

Our team is still working with the Quarkus team to figure out what will be the specific quarkus version (probably Quarkus 2.14.0.Final ).

Just to double-check on dates and timings:

1-) The cut-off for this upgrade will be around November 8th ?
2-) As you are going to PTO, who should we reach out to review/approve the PR with the version changes?

Thanks!